core-bundle/tests/Framework/ContaoFrameworkTest.php

```diff
@@ -650,6 +653,33 @@ public function testServiceIsResetable(): void
         $this->assertNotSame($adapter, $framework->getAdapter(Input::class));
     }
 
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testDelegatesResetCalls(): void
+    {
+        $framework = $this->mockFramework();
+        $framework->setContainer($this->getContainerWithContaoConfiguration());
+        $framework->initialize();
+
+        // setup state in framework classes
+        Environment::set('scriptFilename', 'bar');
+        Input::setUnusedGet('foo', 'bar');
+        $registry = Registry::getInstance();
+        $registry->register(new FooModel());
```

Comment:
Why are you not using an existing model here?

Reply:
Because then the constructor would be called (leading to much more setup). And I cannot mock the model and overwrite static properties/methods like getTable().

Reply:
I don't understand. `new PageModel()` only creates the DCA extract if you do not pass a result object. What "much more setup" are you talking about?

Reply:
So what do I have to do in order for `new PageModel()` to work? For me this throws `Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException : You have requested a non-existent parameter "contao.resources_paths".`